### DISCUSSION TITLE
How to use a local "featured" image in post pages

### DISCUSSION THREAD

**Discussion by darkins** [fylgja/11ty-starter-template#5] [Q&A] [answered]
Hi,

First of all, thanks for this awesome template. I have just started to play around with it and i really like how easy and customizable it is to use. 
I am having the following issues though: 

1.  would like to change the featured images in my posts to my own local images.

Change: 
featured: "https://placeimg.com/640/480/tech"

To: 
featured: "images/my-image.jpg"

My path dir to the images is: src/posts/images

I have also tried placing them here: src/assets/images 
and doing this:
featured: "/images/my-image.jpg" 

I think perhaps my problem my be related to my second issue, which is...

2. How do i successfully run a build?

When i run: 

`npm run build`

I get an error in the terminal saying it doesn't recognise the 'rm' or 'true', so it cannot remove and replace the _site dir.

```
'rm' is not recognized as an internal or external command,
operable program or batch file.
'true' is not recognized as an internal or external command,
operable program or batch file.
```

I would appreciate your help and guidance.

Thanks again for the amazing work you've put into this template!
regards
Declan

**Comment by GrimLink** [comment, marked as answer]
@darkins, Thank you.

Now for your first point;

> would like to change the featured images in my posts to my own local images.

According to the [11ty image plugin documentation](https://www.11ty.dev/docs/plugins/image/#use-this-in-your-templates), the image path must match the entire path.

So use `./src/images/my-image.jpg`.

For the second point;

> How do i successfully run a build?

I'm guessing you're running Windows.

The `rm` command is a UNIX (MacOS, Linux) command that removes the contents of your build folder,
before 11ty will build your site.

It does not work on Windows.

I do intend to replace with for a more cross-platform solution, later.

However, for the time being, replace the `rm -r _site/ || true` command with a Windows solution;

```powershell
# CMD
rmdir /s /q _site

# PowerShell
Remove-Item -Recurse -Force _site
```

I hope this helps 😉

**Reply by darkins** [reply]
Thanks, this did help and yes, i am indeed on windows. 
It would be very nice if it was more cross-platform, as there are a few other issues with the dev process and running cmds. 
There is no way to terminate Ctrl + C on windows and most of the time it doesn't run, when doing `npm start` -  it just hangs after rollup / watching for changes.. and doesn't build a localhost instance.

The template is great though, keep up the good work! 👍
